CHANGELOG.md

```diff
@@ -1,328 +1,711 @@
+* bump dependencies (@ralphtheninja)
+* upgrade to `nan@2.4.0`, remove `Function::NewInstance()` deprecation warnings (@rvagg)
 
-### 0.5.0 @ May 18 2013
+### Removed
+* remove deleted file from `leveldb.gyp` (@juliangruber)
 
-  * Bumped major version for db.getProperty() addition (should have been done in 0.4.4) (@rvagg)
-  * Disallow batch() operations after a write() (@rvagg)
+## [1.4.6] - 2016-04-29
 
-### 0.4.4 @ May 18 2013
+### Added
+* add node 6 to travis (@ralphtheninja)
 
-  * db.getProperty() implemented, see README for details (@rvagg)
-  * More work on memory management, including late-creation of Persistent handles (@rvagg)
+### Changed
+* upgrade to `nan@2.3.0` to remove deprecated calls (@ralphtheninja)
 
-### 0.4.3 @ May 18 2013
+## [1.4.5] - 2016-04-18
 
-  * Better memory leak fix (@rvagg)
+### Added
+* steal chjjs script for checking memory consumption (@ralphtheninja)
 
-### 0.2.2 @ May 17 2013
+### Changed
+* bump dependencies (@ralphtheninja)
+* gitignore Release folder for Windows (@vweevers)
+* update copyright year 2015 -> 2016 (@ralphtheninja)
+* optimize iterators by avoiding handles (@chjj)
 
-  * BACKPORT memory leak fixes (@rvagg)
+### Removed
+* remove unsupported versions from travis (@ralphtheninja)
 
-### 0.4.2 @ May 17 2013
+### Fixed
+* fix delete calls not using `[]` (@chjj)
+* free start slices on conflicting options (@chjj)
+* dispose of unused slices (@chjj)
+* release iterator snapshots (@chjj)
+* fix iterator leak (@chjj)
+* add handlescopes to fix potential memory leaks (@chjj)
+* fix repair-test for multilang windows (@vweevers)
+* repair error is different on windows (@ralphtheninja)
 
-  * Same memory leak fixes as 0.4.1, properly applied to batch() operations too (@rvagg)
+## [1.4.4] - 2016-01-25
 
-### 0.4.1 @ May 17 2013
+### Changed
+* update dependencies (@ralphtheninja)
+* use `prebuild --install` (@ralphtheninja)
+* `README` fixes (@ralphtheninja)
 
-  * Fix memory leak caused when passing String objects in as keys and values, conversion to Slice created new char[] but wasn't being disposed. Buffers are automatically disposed. (@rvagg, reported by @kylegetson levelup/#140)
+### Fixed
+* fix build build problems on windows https://github.com/Level/leveldown/issues/247 (@obastemur)
 
-### 0.4.0 @ May 15 2013
+## [1.4.3] - 2015-12-14
 
-  * Upgrade to LevelDB 1.10.0, fairly minor changes, mostly bugfixes see https://groups.google.com/forum/#!topic/leveldb/O2Zdbi9Lrao for more info (@rvagg)
+### Added
+* `node@5` to travis (@ralphtheninja)
 
-### 0.3.1 @ May 14 2013
+### Changed
+* bump dependencies (@ralphtheninja)
+* add dependency badge (@ralphtheninja)
+* bump `prebuild` for `--all` functionality (@ralphtheninja)
+* prebuild abi 47 (node 5.0) (@ralphtheninja)
 
-  * Don't allow empty batch() operations through to LevelDB, on chained of array forms (@rvagg)
+### Fixed
+* fix build build problems on windows https://github.com/Level/leveldown/issues/196 (@obastemur)
```

Comment:
- build build
- Ah, I see now why the link is useful. Otherwise this item might be mistaken for a duplicate of the above.